package-lock.json

```diff
@@ -48,7 +48,7 @@
       "devDependencies": {
         "@vscode/codicons": "^0.0.36",
         "reflect-metadata": "~0.2.2",
-        "vite": "^6.1.1",
```

Comment:
seems unrelated :wink:

Reply:
renovate does not fully update the package.lock sadly sadly...